lib/cerberus_utils/version.rb

```diff
@@ -1,3 +1,3 @@
 module CerberusUtils
-  VERSION = "2.0.0"
+  VERSION = "2.0.1"
```

Comment:
Should it be more than a x.x.1 release?

Reply:
I think it should. Even though this fixing a bug I think it should at least be a minor release

Reply:
Ok. updated to be a minor version increment